src/utils/i18n.js

```diff
@@ -1,35 +1,86 @@
+import "@babel/polyfill";
 import configs from "./configs";
-import localeData from "../assets/translations.data.json";
+import { FALLBACK_LOCALES } from "../assets/locales/locale_config";
+import defaultLocaleData from "../assets/locales/en.json";
 
-const navigatorLang = ((navigator.languages && navigator.languages[0]) || navigator.language || navigator.userLanguage)
-  .toLowerCase()
-  .split(/[_-]+/)[0];
-export const lang = (localeData[navigatorLang] && navigatorLang) || "en";
+const DEFAULT_LOCALE = "en";
 
-const _messages = localeData[lang] || localeData.en;
-
-if (configs.APP_CONFIG && configs.APP_CONFIG.translations && configs.APP_CONFIG.translations[lang]) {
-  const configTranslations = configs.APP_CONFIG.translations[lang];
-  for (const messageKey in configTranslations) {
-    if (!configTranslations.hasOwnProperty(messageKey)) continue;
-    if (!configTranslations[messageKey]) continue;
-    _messages[messageKey] = configTranslations[messageKey];
+function getLocales() {
```

Comment:
Doesn't need to happen in this PR but it would be good to make available all the languages we have translations for and a language pref in the preferences screen for cases when `navigator.language[s]` doesn't match the user preference.